packages/block-editor/src/components/link-control/settings-drawer.js

```diff
@@ -47,7 +47,7 @@ function LinkSettingsDrawer( {
 				aria-expanded={ settingsOpen }
 				onClick={ () => setSettingsOpen( ! settingsOpen ) }
 				icon={ settingsIcon }
-				label={ __( 'Toggle link settings' ) }
+				label={ __( 'Link Settings' ) }
```

Comment:
👍

Reply:
If this was announced by assistive software wouldn't 'Toggle link settings' be better?

Reply:
Yep exactly that. I'm pretty sure the affordances of the node would be enough but someone more experienced with a11y can confirm.